README.md

```diff
@@ -1,393 +1,6 @@
-
-```
-kubectl crossplane package uninstall --cluster 'crossplane-examples-hello-world'
-```
-
-## How to build for external publishing
-
-After we finish developing a package locally, we may want to publish it to
-an external registry. This section shows the commands to do that.
-
-### Build package
-
-To build the package, we use the `build` subcommand once the binaries and
-generated yamls have been built:
-
-```
-kubectl crossplane package build
-```
-
-### Run publish
-
-Once the package is built, we can use the `publish` subcommand to publish
-it to the registry:
-
-```
-# You may need to log into dockerhub or the docker registry that the
-# image is being pushed to. If that's the case, run:
-# $ docker login
-kubectl crossplane package publish
-```
-
-### Install
-
-Installing the package can be done with a sample which was generated for
-us by the `init` that we ran earlier, but it's a little easier to use
-the `install` command:
-
-```
-kubectl crossplane package install --cluster 'crossplane-examples/hello-world'
-```
-
-### Uninstall
-
-Installing the package can be done with some sample yaml which was
-generated for us by the `init` that we ran earlier, but it's a
-little easier to use the `uninstall` command:
-
-```
-kubectl crossplane package uninstall --cluster 'crossplane-examples-hello-world'
-```
-
-Note that `uninstall` uses the package's name (which has no `/` characters),
-while the `install` uses the image name (which uses `/`).
-
-# Recipes
-
-### Use a different image name when building and testing a package locally
-
-```
-PACKAGE_IMG=myprefix/myothername kubectl crossplane package build
-PACKAGE_IMG=myprefix/myothername kubectl crossplane package publish
-```
-
-### Build locally with an overridden install.yaml
-
-We can specify a different build target to run:
-
-```
-kubectl crossplane package build local-build
-```
-See the `config/package/overrides` directory for details about where the
-overrides live. See the `package.Makefile` for details about how
-`local-build` works.
-
-### Setup RBAC
-
-We can setup extra permissions to grant access to the resources which
-are not part of our package. This can be done by specifying the permissions in the package definition YAML..
-```
-# Human readable title of application.
-title: Sample Wordpress App
-...
-# RBAC Roles will be generated permitting this package to use all verbs on all
-# resources in the groups listed below.
-permissionScope: Namespaced
-dependsOn:
-- crd: "kubernetesclusters.compute.crossplane.io/v1alpha1"
-- crd: "mysqlinstances.database.crossplane.io/v1alpha1"
-- crd: "kubernetesapplications.workload.crossplane.io/v1alpha1"
-...
-```
-For a detailed example, see [here](https://github.com/crossplane/app-wordpress/blob/37443b45f40b73958bfd804f98a2a93ba50d8590/.registry/app.yaml#L60).
-
-[kubebuilder quick start]: https://book.kubebuilder.io/quick-start.html
+This repository and its artifacts are no longer supported by the Crossplane
```

Comment:
What do you think about also linking to the source code for the new CLI, so that folks landing here to see how it works under the hood, or open an issue, or contribute a fix can find the new code base more easily also?

Reply:
@jbw976 I had considered linking to the CLI part of the c/c codebase, but opted not to just to future proof this README against things potentially moving around in the future. That being said, if we did link to the codebase and the link became broken in the future, it wouldn’t be a huge deal as the primary motivation here is to make it such that folks don’t try to install the legacy CLI. I could go either way, what do you think?

Reply:
That reasoning is fine @hasheddan - we can keep it as is in this PR :)